aim/api/resource.py

```diff
@@ -66,6 +85,8 @@ class BridgeDomain(ResourceBase):
     # Attrbutes completely managed by the DB (eg. timestamps)
     db_attributes = []
 
+    _aci_mo_name = 'fvBD'
```

Comment:
Neutron shows DNs for the only purpose of debugability, giving back one part of the story is not very helpful IMHO. An AIM BridgeDomain can't be reconstructed by simply retrieving it's parent object.
But to your point, we can make it a special case of "hidden detail" for all the RS objects, but not all the type of children (like fvSubnet, for instance).

This would be acceptable as long as the RSs' DN can be predicted from the parents' DN. Which is possible with the fvRsCtx case, not sure for rest of the model though.

Lastly, I still believe that using the converter to retrieve the DN (and removing _aci_mo_name) is a good option for separation of concerns, which limits APICAPI presence in the least number of modules (all in the aci_universe package). But this is by no means urgent (and should be fairly easy to do later).